elliott/elliottlib/cli/create_cli.py

```diff
@@ -13,6 +16,50 @@
 from elliottlib.util import YMD, exit_unauthorized, validate_email_address, validate_release_date
 
 LOGGER = logutil.get_logger(__name__)
+GIT_REPO_OWNER = "openshift-eng"
+GIT_REPO_NAME = "ocp-build-data"
+GIT_REPO_BRANCH = "main"
+COMMON_ADVISORY_TEMPLATE_FILE = "config/advisory_templates.yml"
+
+
+def get_common_advisory_template():
+    gh_api = GhApi()
+    content = gh_api.repos.get_content(
```

Comment:
we don't need to use gh api, since we already have build data git repo cloned and ready to use. You can use a method like get_file_from_branch https://github.com/openshift-eng/art-tools/blob/cf5fcf85d7ecb0f06369925a554d4922f6530b18/elliott/elliottlib/runtime.py#L439

Reply:
ack added in https://github.com/openshift-eng/art-tools/pull/1573/commits/588bcdd0118448bef4189750bf0d00936f20122d